docs/usage.md

```diff
@@ -443,6 +443,50 @@ class LineItemViewSet(viewsets.ModelViewSet):
 not render `data`. Use this in case you only need links of relationships and want to lower payload
 and increase performance.
 
+#### Related urls
```

Comment:
Do you think we should combine this documentation with https://django-rest-framework-json-api.readthedocs.io/en/stable/usage.html#related-fields ? 

Maybe we do not need to document overwriting of `get_queryset` anymore as this is obsolete. Or do you still see a use case for it?

Reply:
It is still working and can be used. I think we can remove the docs when we deprecate this stuff.